### ISSUE TITLE
Notification: This extension has been replaced by a fork on the extensions index

### ISSUE THREAD

**Issue by w-e-w** [butaixianran/Stable-Diffusion-Webui-Civitai-Helper#240]
@butaixianran Hi I'm one of the maintainer of [stable-diffusion-webui-extensions](https://github.com/AUTOMATIC1111/stable-diffusion-webui-extensions)

we have received a request over at the extension index to replace the entry with a fork
citing the reason that is extension is no longer functional in the latest version of web UI 
see https://github.com/AUTOMATIC1111/stable-diffusion-webui-extensions/pull/185
some testing and reading through the issue tab does show that this is the case
as such I decide to comply with the request after 1 week if this extension continues to remain in this state

**Comment by w-e-w** [comment]
### one week has passed without receiving any response
this extension has been replaced by a fork on the index

**Comment by pessimal** [comment]
Found this PR because stuff wasn't working. Had to switch to the fork manually because Automatic1111 doesn't do it automatically (?). What I did was:
cd extensions/Stable-Diffusion-Webui-Civitai-Helper
git remote set-url origin https://github.com/zixaphir/Stable-Diffusion-Webui-Civitai-Helper.git
git pull
git checkout master

**Comment by w-e-w** [comment]
yes the is the case
Web UI does not have a system that automatically uninstalls already existing extensions or switching extensions to a fork
we only replaced the extension on index so that new users when installing would install the updated one
> this only applies if you install from the index and not manually input a url

users who have already installed this extension has to manually uninstall the current one

**Comment by butaixianran** [comment]
This extension just catched up with latest sd v1.6. It is fine if sd webui like to replace it with a fork.

**Comment by zixaphir** [comment]
> This extension just catched up with latest sd v1.6. It is fine if sd webui like to replace it with a fork.

Howdy! I just noticed you started picking up development again. I really only started a fork because I had already fixed issues with the latest webui and I felt sharing them would be better than leaving people with a non-op extension. I would be enthused if you would like to merge in my changes and control the canonical branch. I perform the necessary steps to create the merge and send a pull request if you are interested. I was never trying to perform a hostile takeover of your work.
